src/components/Banner/Banner.stories.module.css

```diff
@@ -0,0 +1,3 @@
+.headingBottomSpacing {
+  @apply mb-4;
```

Comment:
Should we maybe be using regular CSS since tailwind is being removed later? Then we won't have to come back and change this.

Reply:
That would make sense for this file since it's just a story file, but for file stylesheets, would have to be changed for tokens anyways. What are your thoughts for this process?

Reply:
For the file stylesheets, could we just leave the existing ones for now and clean it up later when the tokens are in?

Reply:
That sounds good. I also think looking for `@apply` is a good identifier so clean up will be relatively easy